Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/CorelliPowderCalibrationCreate.py

```diff
@@ -0,0 +1,576 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+import numpy as np
+# import os  # Test to see if the build error goes away
+import random
+import string
+from typing import List, Optional, Union
+
+r"""Test to see if the build error goes away
+from corelli.calibration.database import load_calibration_set
+from corelli.calibration.utils import apply_calibration
+"""
```

Comment:
I think we should remove these debugging related comments before merging this PR into master.